dvc/cli.py

```diff
@@ -138,6 +138,16 @@ def get_parent_parser():
     """
     parent_parser = argparse.ArgumentParser(add_help=False)
 
+    # NOTE: We are doing this to capitalize help message.
+    # All subparsers that use this as parent should specify "add_help=False"
```

Comment:
Thanks for the PR! :pray: 

Have you considered adjusting our DvcParser instead of us having to remember to specify `add_help=False` each time? :slightly_smiling_face:

Reply:
yes, I have checked this option, but `add_parser` is part of the implementation (not public interface) [argparse._SubParsersAction](https://github.com/python/cpython/blob/master/Lib/argparse.py#L1122)

Here is an alternative version #4786, please choose one.